docs/guides/src/main/server/logging.adoc

```diff
@@ -155,11 +156,287 @@ You can configure a different logging format for the file log handler by running
 
 Please see the <<Configuring the console log format>> section in this guide for more information and a table of the available pattern configuration.
 
-== Configuring raw quarkus logging properties
-At the time of writing, the logging features of the quarkus based Keycloak are basic, yet powerful. Nevertheless, expect more to come and feel free to join the https://github.com/keycloak/keycloak/discussions/8870[discussion] at GitHub.
+== Centralized logging using Gelf
+Keycloak is able to send logs to a centralized log management system like Graylog, Logstash (inside the Elastic Stack or ELK - Elasticsearch, Logstash, Kibana) or Fluentd (inside EFK - Elasticsearch, Fluentd, Kibana). Keycloak leverages the features of the https://quarkus.io/guides/centralized-log-management[Quarkus Logging Gelf] extension to provide support for these environments.
 
-When you need a temporary solution, e.g. for logging to a file or using syslog instead of console, you can check out the https://github.com/keycloak/keycloak/discussions/8870[Quarkus logging guide]. It is possible to use all properties mentioned there, as long as no other than the base logging dependency is involved. For example it is possible to set the log handler to file, but not to use json output, yet, as you would need to provide another dependency for json output to work.
+=== Enable the Gelf handler
+To enable logging using Gelf, you have to add it to the list of activated log handlers.
 
-To use raw quarkus properties, please refer to the <@links.server id="configuration"/> guide at section _Using unsupported server options_.
+.Example:
+<@kc.start parameters="--log=console,gelf"/>
+
+=== Configure the Gelf handler
+
+To configure the Host and Port of your centralized logging system, run the following command and substitute the values with your specific values:
+.Host and port of the Gelf server:
+<@kc.start parameters="--log=console,gelf --log-gelf-host=myhost --log-gelf-port=12345"/>
+
+By default, when the Gelf handler is enabled, the host is using `localhost` as host value and UDP for communication. If you want to use TCP instead of UDP, prefix the host value with `tcp:`. The Default port is `12201`.
+
+. Gelf protocol version
+By default, Keycloak uses Gelf version `1.1`. If you need to use version 1.0 instead, run the following command:
+
+<@kc.start parameters="--log=console,gelf --log-gelf-version=1.0"/>
```

Comment:
Wondering if we can avoid adding `log-gelf-version` and have handlers like `gelf`, `gelf-1`, `gelf-1.1`. Where `gelf` is a shorthand for the default version.

Looks simpler to me.